ng/src/web/pages/tags/component.js

```diff
@@ -167,25 +167,35 @@ class TagComponent extends React.Component {
     if (is_defined(tag)) {
       const {resource = {}} = tag;
       this.setState({
+        active: tag.active,
+        comment: tag.comment,
         dialogVisible: true,
+        fixed: tag.fixed,
```

Comment:
I don't think fixed is a property of the tag. I guess fixed is used for displaying the dialog with a fixed part. Could you check this? Maybe I am wrong.

Reply:
That seems to be the case, indeed. I changed that and don't put fixed into the state anymore.